api/agent/drivers/docker/docker_client.go

```diff
@@ -119,11 +226,10 @@ func (d *dockerWrap) retry(ctx context.Context, logger logrus.FieldLogger, f fun
 		if common.IsTemporary(err) || isDocker50x(err) {
 			logger.WithError(err).Warn("docker temporary error, retrying")
 			b.Sleep(ctx)
-			span.LogFields(log.String("task", "tmperror.docker"))
```

Comment:
would be good to still track these? but ok...

Reply:
i think they are duplicates of the retries count sum. the terminal cases (non-temporary error, timeout) have explicit stats and any non-zero retries count would sum to the same as this previous `tmperror`. can add back tho?

Reply:
I don't know, if docker starts to spits 50x, etc. it would be good to see, but we can tweak later too...

Reply:
i am positive that it's encompassed in the retries sum a little higher up (this was also a sum)